portal-api/src/v1/gef/controllers/application.controller.js

```diff
@@ -246,11 +249,12 @@ exports.changeStatus = async (req, res) => {
   let applicationUpdate = { status, ...{ updatedAt: Date.now() } };
 
   if (status === DEAL_STATUS.SUBMITTED_TO_UKEF) {
-    const submissionData = await addSubmissionData(dealId, existingApplication);
+    const submissionData = await addSubmissionData(dealId, existingApplication, generatePortalAuditDetails(req.user._id));
 
     applicationUpdate = {
       ...applicationUpdate,
       ...submissionData,
+      auditRecord: generatePortalUserAuditDatabaseRecord(req.user._id),
```

Comment:
Sorry big pain thing -- if audit details change later, we might miss updating here. To solve this, could we do the below please?

```suggestion
    const auditDetails = generatePortalAuditDetails(req.user._id);
    ...
    const submissionData = await addSubmissionData(dealId, existingApplication, auditDetails);

    applicationUpdate = {
      ...applicationUpdate,
      ...submissionData,
      auditRecord: generatePortalUserAuditDatabaseRecord(auditDetails),
```